git-hub

```diff
@@ -6,7 +6,69 @@ import click
 import sys
 import textwrap
 import json
+import toml
 import os
+from urllib.request import urlopen, Request
+
+from os.path import join as pjoin
+
+def login():
+    """Retrieves the user's remote.
+
+    This will run the command "git remote -v",
+    then abstract the username and repo from the remote.
+
+    Returns
+    -------
+    username : string
+        username of the remote
+    repo : string
+        repository of the remote
+
+    """
+    process = subprocess.Popen(["git", "remote", "-v"], stdout=subprocess.PIPE)
+    remotes = str(process.stdout.read())
+    url = remotes.split(" ", 1)[0]  # gets the fetch url
+    arguments = url.split(".com")[1]  # gets just the username/repo.git
+    arguments = arguments[1:]
+    arguments = arguments.split(".git")[0]  # takes out ".git"
+    username, repo = arguments.split("/")
+    return (username, repo, remotes)
+
+
+def get_token():
+    """Will get token from authentification file
+
+    Returns
+    -------
+    token : string
+        a number corresponding to user's
```

Comment:
Wrap at 80 characters.  Spelling mistake.

Make it something like "GitHub token, used to push changes."